redbeams/src/main/java/com/sequenceiq/redbeams/converter/database/DatabaseV4RequestToDatabaseConfigConverter.java

```diff
@@ -0,0 +1,60 @@
+package com.sequenceiq.redbeams.converter.database;
+
+
+import static com.sequenceiq.cloudbreak.common.type.APIResourceType.RDS_CONFIG;
+
+import java.util.Date;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.google.common.base.Strings;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.DatabaseVendor;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.ResourceStatus;
+import com.sequenceiq.cloudbreak.common.converter.MissingResourceNameGenerator;
+import com.sequenceiq.cloudbreak.converter.AbstractConversionServiceAwareConverter;
+import com.sequenceiq.redbeams.api.endpoint.v4.database.request.DatabaseV4Request;
+import com.sequenceiq.redbeams.domain.DatabaseConfig;
+import com.sequenceiq.redbeams.util.DatabaseVendorUtil;
+
+@Component
+public class DatabaseV4RequestToDatabaseConfigConverter  extends AbstractConversionServiceAwareConverter<DatabaseV4Request, DatabaseConfig> {
```

Comment:
Do you have a unit test for this converter? If not, that's fine, we can add one later. I'd rather not wait and get this good stuff merged in.

Reply:
Ok, I added a jira task.